src/EnergyPlus/DXCoils.cc

```diff
@@ -7500,8 +7492,8 @@ namespace DXCoils {
 				OutdoorWetBulb = OutWetBulbTemp;
 			}
 			if ( DXCoil( DXCoilNum ).IsSecondaryDXCoilInZone ) {
-				OutdoorDryBulb = Node( DXCoil( DXCoilNum ).SecZoneAirNodeNum ).Temp;
-				OutdoorHumRat = Node( DXCoil( DXCoilNum ).SecZoneAirNodeNum ).HumRat;
+				OutdoorDryBulb = ZT( DXCoil( DXCoilNum ).SecZonePtr );
+				OutdoorHumRat = ZoneAirHumRat( DXCoil( DXCoilNum ).SecZonePtr );
 				OutdoorWetBulb = DXCoil( DXCoilNum ).EvapInletWetBulb;
 				OutdoorPressure = OutBaroPress;
 			}
```

Comment:
Wait a second, I can't do that either since node data is not written there now (NodeNum=0). So even providing a node number may require other changes to write the data to that new node number. I'd have to check to see if only controlled zones write the data to the zone air node but it makes sense that would be the case.